src/main/java/cloud/eppo/ufc/dto/adapters/BanditParametersResponseDeserializer.java

```diff
@@ -11,21 +11,38 @@
 import java.util.HashMap;
 import java.util.Iterator;
 import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BanditParametersResponseDeserializer
+    extends StdDeserializer<BanditParametersResponse> {
+  private static final Logger log =
+      LoggerFactory.getLogger(BanditParametersResponseDeserializer.class);
 
-public class BanditsDeserializer extends StdDeserializer<Map<String, BanditParameters>> {
   // Note: public default constructor is required by Jackson
-  public BanditsDeserializer() {
+  public BanditParametersResponseDeserializer() {
     this(null);
   }
 
-  protected BanditsDeserializer(Class<?> vc) {
+  protected BanditParametersResponseDeserializer(Class<?> vc) {
     super(vc);
   }
 
   @Override
-  public Map<String, BanditParameters> deserialize(
+  public BanditParametersResponse deserialize(
```

Comment:
We now return the whole response instead of just handling the `bandits` property of it to make this file more analogous to `FlagConfigResponseDeserializer`